examples/elastic-header/style.css

```diff
@@ -1,6 +1,5 @@
 h1 {
-  font-weight: 300;
-  font-size: 1.8em;
+  font: 300 1.8em Arial, sans-serif;
```

Comment:
What's wrong with the original declarations? There's no need to define `font-family` here.

Reply:
In an actual live implementation of the code we can assume that a font-familly would defined. The current code omits this and leaves the font-family of the h1 tag to be determined by the browser.

e.g.
font-weight: 300;
font-size: 1.8em;
font-family: x;

Thus, we can shorten future logic into:
font: 300 1.8em x;

Reply:
This is an example, and its key point is to demonstrate how to create things with Vue. In my opinion this PR is off-topic whatsoever, because it doesn't have much to do with that purpose.

Reply:
Omitting a property can be intentional. That’s why we have longhand properties.